### ISSUE TITLE
[Feature Suggestion] Timer Ranges

### ISSUE THREAD

**Issue by SeanMirrsen** [gta-chaos-mod/ChaosModV#563]
Stemming from the discussion about metaeffects on Discord, was the observation that the delay between metaeffect activations is a little too uniform, and it could stand to be different for different effects. It was also noticed that 'Chaos' mod is itself still suffering from a bit of Order, in that its timer lengths are constant, outside of metaeffects modifying them.

With that in mind, the suggestion is to implement Timer Ranges. Right now the main timer is a fixed value, defaulting to 30 seconds. What this suggestion would do, is allow both the main and metaeffect timers, to be defined as a range, from a lower to higher bound - say 20 to 40 seconds for the main timer. The metaeffect timer could maybe be defined between 8 and 12 minutes rather than the default 10.

Duration ranges for effects could possibly also be implemented, to make the effects more varied in length than just 'long' and 'short'.

**Comment by anto4440** [comment]
No. One of the most important parts of chaos is that the timer is fixed. This idea would work better as an option rather than the new system. Apologies if that was what you meant

**Comment by SeanMirrsen** [comment]
I mean. If you set top and bottom of the range as the same value it'll be exactly like the current system. So it's an option by default. :P
